tests/cypress/e2e/contentEditor/hidePreview.cy.ts

```diff
@@ -0,0 +1,91 @@
+import {addNode, createSite, deleteSite, getNodeByPath} from '@jahia/cypress';
+import {JContent} from '../../page-object';
+import {RepositoryExplorer} from '../../page-object/repositoryExplorer';
+import {CategoryManager} from '../../page-object/categoryManager';
+
+const siteKey = 'hidePreviewSite';
+describe('Hide Preview testsuite', () => {
+    before('Create site and content', () => {
+        createSite(siteKey);
+        addNode({
+            parentPathOrId: `/sites/${siteKey}/contents`,
+            name: 'ContentFolder',
+            primaryNodeType: 'jnt:contentFolder'
+        });
+        addNode({
+            parentPathOrId: `/sites/${siteKey}/contents`,
+            name: 'Text',
+            primaryNodeType: 'jnt:bigText'
+        });
+        addNode({
+            parentPathOrId: `/sites/${siteKey}/files`,
+            name: 'Folder',
+            primaryNodeType: 'jnt:folder'
+        });
+    });
+
+    beforeEach('login and visit home', () => {
+        cy.login();
+        const jcontent = new JContent();
```

Comment:
you should avoid creating JContent object (or any page object) and get it with its static accessor (like visit)